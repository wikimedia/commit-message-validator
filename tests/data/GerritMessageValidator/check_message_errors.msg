XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
YYYYYYYYYYY
YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY
Fixes:foo
Closes:bar
Task: T11
BUG:13
Bug: T11 T12

Bug: T11
Bug: T12
Bug: t13

Depends-On: I395fd614bfa8bcfc46a35f955fb77ec6f03f7a01

Bug: t14
Depends-On: I00d0f7c3b294c3ddc656f9a5447df89c63142203
Change-Id: I20c70ff250085de49e4b961da4b53258858df1dd
Change-Id: I62a5bbf5a94da6f121a7836859371a2a2f60873d
Depends-On: I9abc

Foo bar
(cherry picked from commit 7ad8cc24d0672111cff5beddcfc9d778a1a3478e)
