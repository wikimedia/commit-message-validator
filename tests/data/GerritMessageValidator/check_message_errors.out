The following errors were found:
- Line 1: Subject must be <=80 characters
- Line 2: Second line should be empty
- Line 3: Line exceeds max length (174>100)
- Line 4: Expected 'Fixes:' to be in footer
- Line 4: Use 'Bug:' not 'Fixes:'
- Line 4: Expected one space after 'Fixes:'
- Line 4: Bug: value must be a single phabricator task ID
- Line 5: Expected 'Closes:' to be in footer
- Line 5: Use 'Bug:' not 'Closes:'
- Line 5: Expected one space after 'Closes:'
- Line 5: Bug: value must be a single phabricator task ID
- Line 6: Expected 'Task:' to be in footer
- Line 6: Use 'Bug:' not 'Task:'
- Line 7: Expected 'BUG:' to be in footer
- Line 7: Use 'Bug:' not 'BUG:'
- Line 7: Expected one space after 'BUG:'
- Line 7: Bug: value must be a single phabricator task ID
- Line 8: Expected 'Bug:' to be in footer
- Line 8: Bug: value must be a single phabricator task ID
- Line 12: Bug: value must be a single phabricator task ID
- Line 13: Unexpected blank line
- Line 15: Unexpected blank line
- Line 16: Bug: value must be a single phabricator task ID
- Line 19: Extra Change-Id found, first at 18
- Line 20: Depends-On: value must be a single Gerrit change id
- Line 21: Unexpected blank line
- Line 22: Expected footer line to follow format of 'Name: ...'
